code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c

```diff
@@ -0,0 +1,66 @@
+/* Check weather element is present in link list (iterative method)*/
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+struct node{                /* defining a structure named node */
```

Comment:
As an example here, the spacing isn't correct for the line above, and you don't need newlines between the data members.